sdb/commands/array.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+import argparse
+from typing import Iterable
+
+import drgn
+import sdb
+
+class Array(sdb.Command):
+    # pylint: disable=too-few-public-methods
+
+    names = ["array"]
+
+    def _init_argparse(self, parser: argparse.ArgumentParser) -> None:
+        # TODO allow 'count' to be omitted if input type is proper array
+        parser.add_argument("count", type=int, metavar="<count>")
```

Comment:
I'm OK leaving this for a future change (i.e. feel free to drop this), but it might be a useful addition to enable a range to be supplied here, rather than simply the first N entries.. and perhaps use the python syntax for the `[]` operator.. e.g. `:N`, `N:`, `:-1`, etc.

Reply:
This might actually be better to be implemented by the `index` command.. and then this `array` command might be unnecessary.

I don't expect you to make any changes, but just writing down my thoughts as I'm looking over this.